index.js

```diff
@@ -52,10 +48,22 @@ const employees = {
   "009": "Thejasri",
 };
 
+//punch in landing page method:
+app.get("/", (_, res) => {
+  return res.render("punchIn", {
+    title: "PunchIn/Out-system",
+  });
+});
+
 //punch in submit method:
 app.post("/", async (req, res) => {
   //if data is empty it will not store the data into db instead it will redirect to the home page
-  if (req.body.id === "") return res.redirect("/");
+  if (
+    req.body.id === "" ||
```

Comment:
for all user input validations, use trim method
this will eliminate white spaces